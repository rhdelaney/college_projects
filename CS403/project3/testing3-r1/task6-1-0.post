(println "trying (pfs 3 23)")
(stream-display 23 (apply pfs '(3 23)))
(println)
