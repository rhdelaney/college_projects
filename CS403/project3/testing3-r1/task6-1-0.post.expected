trying (pfs 3 23)
(3 9 23 27 69 81 207 243 529 621 729 1587 1863 2187 4761 5589 6561 12167 14283 16767 19683 36501 42849 ...)
